Aquarius/Aquarius.cpp

```diff
@@ -27,6 +30,16 @@ int Test::testMain()
 
     AQ_INFO("Window created successfully!");
 
+    //TODO Bring in neccesary set methods for window events.
+    glfwSetKeyCallback(window, KeyboardEvent::Callback);
+    glfwSetCursorPosCallback(window, MouseEvent::cursorPositionCallback);
+    glfwSetCursorEnterCallback(window, MouseEvent::cursorInWindowCallback);
+    glfwSetMouseButtonCallback(window, MouseEvent::mouseButtonCallback);
+    glfwSetScrollCallback(window, MouseEvent::mouseScrollCallback);
+
+    glfwSetInputMode(window, GLFW_STICKY_MOUSE_BUTTONS, 1);
+    glfwSetInputMode(window, GLFW_STICKY_KEYS,1);
```

Comment:
All this should eventually happen within some of our internal code, either in the application or window initialization I would say

Reply:
Good call, we can discuss the best spot for this.